Makefile

```diff
@@ -95,11 +97,13 @@ endif
 
 BASEUBIIMAGE ?= registry.access.redhat.com/ubi8/ubi-minimal:8.4
 
+OPERATORIMAGENAME ?= $(REPONAME)
```

Comment:
Is this overwritten for enterprise?

Reply:
Yes, release-manager will set it for enterprise to (kube-arangodb-enterprise or kube-arangodb-enterprise-preview)

Reply:
In make we just set default value to be able to build everything using old methods